java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/LspAnalyser.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.java.lsp.server.protocol;
+
+import org.netbeans.api.annotations.common.NonNull;
+import org.netbeans.modules.java.lsp.server.ui.AnalyserConfig;
+import org.netbeans.modules.java.source.usages.BinaryAnalyser;
+import org.openide.util.lookup.ServiceProvider;
+
+/**
+ *
+ * @author Dusan Balek
+ */
+@ServiceProvider(service = BinaryAnalyser.Config.class, position = 1000)
+public final class LspAnalyser extends BinaryAnalyser.Config {
+    private static final String PROP_USG_LVL = "org.netbeans.modules.java.source.usages.BinaryAnalyser.usages"; //NOI18N
+    private static final String PROP_ID_LVL = "org.netbeans.modules.java.source.usages.BinaryAnalyser.idents"; //NOI18N
```

Comment:
It's quite unfortunate these properties need to be copied into `java.lsp.server` module. I don't have a better advice yet, but I am thinking about it.

Reply:
The easiest solution would be to really use branding API as suggested by [comment](https://github.com/apache/netbeans/pull/2950#pullrequestreview-659995669).

Reply:
OK. Branding API used as suggested.